src/language/branches.md

```diff
@@ -129,20 +129,14 @@ check out the patterns documentation on [Switch statements and expressions][].
 
 ### Switch expressions
 
-A `switch` expression is similar to a `switch` statement, but you can use them
-anywhere you can use an expression. They produce a value based on the expression
-body of whichever case matches.
+A `switch` _expression_ produces a value based on the expression
+body of whichever case matches. 
+You can use a switch expression wherever Dart allows expressions,
+_except_ at the start of an expression statement.
+If your switch doesn't need to return a value into some expression statement,
+use a [switch _statement_](#switch-statements) instead of an expression.
```

Comment:
**nit:** This is a little long, but could go as is.
```suggestion
Not all switch statments need to return a value into some expression statement.
If yours doesn't, use a [switch _statement_](#switch-statements) rather than an expression.
```

Reply:
Thanks Tony! Sounds good